app/controllers/organizations_controller.rb

```diff
@@ -1,20 +1,12 @@
 class OrganizationsController < ApplicationController
-  before_filter :load_resource
-
-  def load_resource
-    if params[:id]
-      @organization = Organization.find(params[:id])
-    else
-      @organizations = Organization.all
-    end
-  end
+  before_filter :load_resource, except: [:new, :index, :create]
 
   def new
     @organization = Organization.new
   end
 
   def index
-    @organizations = @organizations.matching(params[:q]) if params[:q].present?
```

Comment:
why are we removing the `q` param? That will break the search, right?

Reply:
I deleted it becasue I can't found any point of the app using this search.